src/actions/action.ts

```diff
@@ -92,6 +92,7 @@ export function createActions(menuItems: ActionSpec[]): Action[] {
 }
 
 const uuidGenerator = new UuidGenerator();
+uuidGenerator.setIsFastStrategy(true);
```

Comment:
let's just use a incremented number here then. Instead of instantiating a `UuidGenerator` for no apparent reason.
Or it's the beginning of the [enterprise fizzbuzz](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition) :smile: